src/include/network/postgres_protocol_handler.h

```diff
@@ -198,26 +197,6 @@ class PostgresProtocolHandler: public ProtocolHandler {
   std::unordered_map<std::string, stats::QueryMetric::QueryParamBuf>
       statement_param_types_;
 
-  //TODO: should this stay in traffic_cop?
-  std::shared_ptr<Statement> statement_;
-
-  std::string error_message_;
```

Comment:
Error_message should probably stay in protocol handler, since it is protocol related

Reply:
I think the Error_message is created when exception threw in prepareStatement and executeStatement in traffic_cop. So I just save it like statement_ in traffic_cop

Reply:
I see,
Then this is good.